tests/framework/microvm.py

```diff
@@ -456,6 +456,22 @@ def pin_api(self, cpu_id: int):
             return True
         return False
 
+    def pin_threads(self, first_cpu):
+        """
+        Pins all microvm threads (VMM, API and vCPUs) to consecutive physical cpu core, starting with "first_cpu"
+        """
+        for i in range(self.vcpus_count):
```

Comment:
Shouldn't this depend on first_cpu? Since you have to keep track of both vcpus and pcpus, we may need sthg like:

```python
first_cpu = 8
vcpu_count = 4

for vcpu, pcpu in enumerate(range(first_cpu, first_cpu + vcpu_count)): 
    print(vcpu, pcpu)
0 8
1 9
2 10
3 11
```

Reply:
ahhh, yes, good catch! Fixed!